frontend/workflows/projectSelector/src/dash.tsx

```diff
@@ -32,15 +41,32 @@ const dashReducer = (state: DashState, action: DashAction): DashState => {
   }
 };
 
+const timelineReducer = (state: TimelineState, action: TimelineAction): TimelineState => {
+  switch (action.type) {
+    case "UPDATE": {
+      // for now, clobber any existing data
+      const newState = { ...state };
+      newState.timeData[action.payload.key] = action.payload.points;
+      return newState;
+    }
+    default:
+      throw new Error("not implemented (should be unreachable)");
+  }
+};
+
 const Dash = ({ children }) => {
   const [state, dispatch] = React.useReducer(dashReducer, initialState);
-
+  const [timelineState, timelineDispatch] = React.useReducer(timelineReducer, initialTimelineState);
   return (
     <Box display="flex" flex={1} minHeight="100%" maxHeight="100%">
       <DashDispatchContext.Provider value={dispatch}>
         <DashStateContext.Provider value={state}>
-          <ProjectSelector />
-          <CardContainer>{children}</CardContainer>
+          <TimelineDispatchContext.Provider value={timelineDispatch}>
+            <TimelineStateContext.Provider value={timelineState}>
```

Comment:
we can figure this out later but if we want the project selector contexts to be modified by the timeline we should change the ordering here. I don't think there is a decision on this yet though.